docs/topics/Rate_Limits.md

```diff
@@ -26,7 +26,21 @@ X-RateLimit-Reset: 1470173023
 
 ## Exceeding A Rate Limit
 
-In the case that a per-route rate limit is exceeded, we do return an HTTP 429 response from our server. It looks something like the following[:](http://takeb1nzyto.space/)
+In the case that a per-route rate limit is exceeded, we do return an HTTP 429 response from our server.
```

Comment:
Should the "per-route" part be dropped, as this applies to both the per-route and global rate limits?

> In the case that the rate limit is exceeded, the API will return a HTTP 429 response code with a JSON body.

Or maybe note that both types return the same type of response?

> In the case that either a per-route or global rate limit is exceeded, the API will return a HTTP 429 response code with a JSON body.